force-app/main/default/classes/RecurrenceService_TEST.cls

```diff
@@ -119,6 +119,7 @@ public with sharing class RecurrenceService_TEST {
         String daysMethod = 'withDays';
         String resetMethod = 'reset';
         String buildMethod = 'build';
+        String byDay = '+1MO';
```

Comment:
I think this shouldn't have the `+` at this point?

Reply:
copy/paste error. Will remove that. Thanks for catching it